src/client/components/Sidebar/InterestingPeople.less

```diff
@@ -7,7 +7,7 @@
 }
 
 .InterestingPeople__container {
-  padding: 8px 16px 16px 16px;
+  transform: scale(.9);
```

Comment:
why this and remove the current padding? The padding is similar to all our other sidebar components.

Reply:
padding take more space instead of scale.